modules/ossm-tutorial-bookinfo-install.adoc

```diff
@@ -7,7 +7,7 @@ This PROCEDURE module included in the following assemblies:
 [id="ossm-tutorial-bookinfo-install_{context}"]
 = Installing the Bookinfo application
 
-This tutorial walks you through how to create a Bookinfo project, deploying the Bookinfo application, and run Bookinfo on {product-title} with {ProductShortName} {ProductVersion}.
+This tutorial walks you through how to create a sample application by creating a Bookinfo project, deploying the Bookinfo application, and running Bookinfo on {product-title} with {ProductShortName} {ProductVersion}.
```

Comment:
Do users benefit from 2 descriptive gerund phrases (deploying... and running...) if the procedure below then repeats the same information?

If this chapter describes a 1-out-of-many way of installing the Bookinfo app then indeed it might be useful to point out that this procedure uses Service Mesh (unlike others). Will it make sense to add it to the title them?

If this is the only one way to install the Bookinfo app, maybe the introductory phrase could be shortened? Given that users tend to skip directly to procedure steps with commands...

Something like this could work maybe: This tutorial shows how to create a sample application by creating a Bookinfo project.

WDYT?

Reply:
The project/namespace and application are two different things that are created separately.  It's basically sample data that we provide with service mesh (because a service mesh monitors microservice applications) as an example.

Reply:
I revised this section to clarify.

Reply:
I see now, thank you for explaining.

The only question left: whether we need a comma between 2 gerund phrases: deploying ... and running... In theory they're equal like "oranges and apples" but maybe an extra comma here helps with readability?